### PR TITLE
Update PSR container to 2.0.2 to match with server

### PR DESCRIPTION
**Pull request by susnux**
Update after nextcloud/server#36617 is merged

### CODE REVIEW

**Comment by susnux** [comment]
Ping @nickvergessen @ChristophWurst

**Comment by nickvergessen** [comment]
Also was backported, iirc?

File: composer.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
     "require": {
         "php": "^7.4 || ~8.0 || ~8.1",
```
Comment (nickvergessen) [inline composer.json:13]:
```suggestion
        "php": "~8.0 || ~8.1 || ~8.2",
```

**Review by nickvergessen — COMMENTED** [review]
But can also do a second PR

**Comment by susnux** [comment]
@nickvergessen 
> Also was backported, iirc?

Only master + NC27. 26 and 25 only got a version bump to 1.1.2

File: composer.json (hunk 1 above)
Comment (susnux) [reply composer.json]: @nickvergessen fixed!

**Comment by susnux** [comment]
> @nickvergessen
> 
> > Also was backported, iirc?
> 
> Only master + NC27. 26 and 25 only got a version bump to 1.1.2

**Comment by susnux** [comment]
Should have been a comment about the companion PR:
https://github.com/nextcloud-deps/ocp/pull/9

File: composer.json [hunk 2]
```diff
@@ -10,11 +10,11 @@
...
+        "php": "~8.0 || ~8.1 || ~8.2",
         "psr/clock": "^1.0",
-        "psr/container": "^1.1.1",
+        "psr/container": "^2.0.2",
         "psr/event-dispatcher": "^1.0",
-        "psr/log": "^1.1"
+        "psr/log": "^1.1.4"
     },
     "extra": {
         "branch-alias": {
```
[the remark this answers is not in the document]
Comment (susnux) [reply composer.json:20]: Already filed a PR for that change 🙈 
https://github.com/nextcloud-deps/ocp/pull/10

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**